frontend/pages/event/create.vue

```diff
@@ -0,0 +1,44 @@
+<script setup lang="ts">
+import { LocalStorageEnum } from '~/constants/enums/common';
+import type { PostEventPayload } from '../../../common/types/event';
+import { useEventStore } from '~/stores/event.store';
+
+const router = useRouter();
+const localePath = useLocalePath();
+
+const eventStore = useEventStore();
+
+onMounted(async () => {
+	await eventStore.getTimezones();
+});
+
+const initialValues = getInitialEventFormValues();
+
+const submitEvent = async (payload: PostEventPayload) => {
+	try {
+		const { data } = await apiRouter.events.add.useMutation({
+			data: { event: payload }
+		});
+		localStorage.removeItem(LocalStorageEnum.EVENT_DATA);
+
+		await navigateTo(localePath(`/event/${data.value.id}`));
```

Comment:
Лучше использовать `RoutePathEnum` из `frontend/constants/enums/route.ts`. Это позволит избежать "волшебных" строк и менять строки (в случае необходимости) в единственном месте для всего приложения
```suggestion
		await navigateTo(localePath(RoutePathEnum.EVENT, '/', data.value.id));
```

Reply:
fixed хотя я скопипастила из другой страницы